requirements.txt

```diff
@@ -56,6 +56,7 @@ snowflake-connector-python==2.7.9
 sshtunnel==0.4.0
 tables==3.7.0
 trino==0.321.0
+-e git+https://github.com/mage-ai/google-ads-python#egg=google-ads
```

Comment:
Can you add this to the requirements.txt under `mage_integrations` module?

Reply:
Fixed on commit bd83786638ad42f1c9a6ce0cef1916c37fe312ce